scripts/gen-env-files.sh

```diff
@@ -95,7 +95,7 @@ if test -f "./frontend/.env"; then
 else
 echo \
 "NODE_ENV=development
-API_URL=http://backend:8080/
+API_URL=http://localhost:5000/
```

Comment:
@asanchezr @devinleighsmith do you know if the gen configuration is used by non development? otherwise this should be fine

Reply:
same comment as above